BackendBench/suite/torchbench.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, name, filename=None, filter=None, topn=None):
             format="auto",  # Auto-detect based on file extension
             filter=filter,
         )
+        if performance_canary_mode:
```

Comment:
Still kinda confused by the naming, wouldn't something like `exclude_mostly_overhead_ops` be more descriptive